src/app/components/pages/admin-cell/admin-cell.component.html

```diff
@@ -1,5 +1,96 @@
 <div class="admin-cell" *ngIf="cell">
-  <h2>Cellule de {{ cell.name }}</h2>
+  <div class="admin-cell__title">
+    <h2>Cellule de {{ cell.name }}</h2>
+
+  </div>
+
+  <app-my-modal name="create event"
```

Comment:
It's a common practice to put all modals at the end of the HTML file, it's more easy to read the code (and CSS structure) and it's more easy to find out if the page have a modal or not.

You just needt to cut-paste this one at the end of the file, it would be work directly since the button is linked with a service.